--- apitrace-14.0/thirdparty/CMakeLists.txt.orig	2026-01-30 16:43:06.000000000 +0100
+++ apitrace-14.0/thirdparty/CMakeLists.txt	2026-04-19 15:08:41.316463297 +0200
@@ -16,26 +16,6 @@ function (include_with_scope)
     include (${ARGV})
 endfunction ()
 
-set (SUBMODULES_MISSING FALSE)
-foreach (path IN ITEMS
-    brotli/LICENSE
-    gtest/LICENSE
-    libbacktrace/LICENSE
-    libpng/LICENSE
-    snappy/COPYING
-    zlib/README
-    zstd/LICENSE
-    directxmath/LICENSE
-)
-    if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${path}")
-        message (SEND_ERROR "error: ${CMAKE_CURRENT_SOURCE_DIR}/${path} does not exist")
-        set (SUBMODULES_MISSING TRUE)
-    endif ()
-endforeach ()
-if (SUBMODULES_MISSING)
-    message (FATAL_ERROR "Update Git submodules by running\ngit submodule update --init --depth 1 --recursive")
-endif ()
-
 if (ENABLE_STATIC_SNAPPY OR NOT Snappy_FOUND)
     message (STATUS "Using bundled Snappy")
     include_with_scope (snappy.cmake)
@@ -83,10 +63,6 @@ endif()
 
 target_optimize(zstd_seekable)
 
-if (CMAKE_EXECUTABLE_FORMAT STREQUAL "ELF")
-    include_with_scope (libbacktrace.cmake)
-endif ()
-
 if (BUILD_TESTING)
     # We use non-standard C++ flags, so we can't just use GTest's CMakeLists.txt
     if (NOT GTEST_FOUND)
